x-pack/plugins/observability/public/embeddable/slo/overview/slo_overview_grid.tsx

```diff
@@ -133,6 +136,15 @@ export function SloCardChartList({ sloId }: { sloId: string }) {
       sloTarget,
       historicalSummary,
     });
+    data.body = (
```

Comment:
In slo_overview file you removed any reference to the containerRef, but this file still uses it. I guess it is a leftover. Can you remove any reference to containerRef in this file?